### PR TITLE
[READY] Tile Reskinning 2: Epic Winaloo

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

extends the tile reskinning functionality to iron, bronze, plastitanium, carpet and pod floors
makes a bit of tile code better
moves some paths around, like elevator shafts being plating instead of floor
adds rotating as a tile reskinning function available on chapel or side floors for example
image

- [x] fix radials
- [x] fix maps
- [x] get sprites

## Why It's Good For The Game

lets players customize any rooms they would want much more than it is possible now, allowing for more creativity

## Changelog
:cl: Fikou, TWATICUS
qol: You can now reskin a lot more tiles! Use iron, bronze, pod, carpet or plastitanium tiles inhand to change their sprites.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
Waiting for sprites, code can be reviewed now.

**Comment by maxymax13** [comment]
god motherfucking bless you

**Comment by maxymax13** [comment]
the original skyrat version of the original tile floofing pr has some iron tile sprites 
https://github.com/Skyrat-SS13/Skyrat13/pull/2838
image

**Comment by Fikou** [comment]
> the original skyrat version of the original tile floofing pr has some iron tile sprites

those are shiny and based on titanium floors, twaticus is making some epic ones

**Comment by AffectedArc07** [comment]
This PR crashed mapdiffbot
image
image

Repo suspended until this PR is dealt with

**Comment by maxymax13** [comment]
pr too swag for dumb robot

File: code/game/turfs/open/floor/iron_floor.dm [hunk 1]
```diff
@@ -209,28 +198,27 @@
...
-
-
-/turf/open/floor/iron/elevatorshaft
-	icon_state = "elevatorshaft"
-	base_icon_state = "elevatorshaft"
-
 /turf/open/floor/iron/bluespace
 	icon_state = "bluespace"
 	base_icon_state = "bluespace"
+	desc = "Sadly, these don't seem to make you faster..."
```
Comment (maxymax13) [inline code/game/turfs/open/floor/iron_floor.dm:204]: silly you, it's red that makes things go faster, not blue, smh basic science

**Comment by Fikou** [comment]
someone retsart checks please

**Comment by maxymax13** [comment]
checks have gone on strike on protest of your pr violently murdering mapdiffbot

**Comment by maxymax13** [comment]
what happened to selecting jesus carpets?

**Comment by Fikou** [comment]
> what happened to selecting jesus carpets? there was an image and then you removed it

nothing it still exists

**Comment by RaveRadbury** [comment]
This is heroic.

Thank you for making it.

**Review by Twaticus — APPROVED** [review]

**Comment by KathrinBailey** [comment]
my baby is all grown up

**Pull request merged**